src/components/victory-candlestick/victory-candlestick.js

```diff
@@ -455,10 +467,12 @@ export default class VictoryCandlestick extends React.Component {
   }
 
   render() {
+    const modifiedProps = this.props.theme ? Helpers.modifyProps(this.props, fallbackProps,
+      this.props.theme.candlestick.props) : Helpers.modifyProps(this.props, fallbackProps);
```

Comment:
Oh, never mind. I see that you need the theme scoped to `candlestick`. But for me, the demo is failing on this line since `this.props.theme.candlestick.props` doesn't protect for  `this.props.theme.candlestick` being undefined